eng/pipelines/templates/stages/archetype-sdk-client.yml

```diff
@@ -0,0 +1,19 @@
+parameters:
+  Artifacts: []
+  ServiceDirectory: not-specified
+  
+stages:
+  - stage: Build
+    jobs:
+    - template: ../jobs/archetype-sdk-client.yml
+      parameters:
+        ServiceDirectory: ${{parameters.ServiceDirectory}}
+        Artifacts: ${{parameters.Artifacts}}
```

Comment:
The job template doesn't declare the parameter `Artifacts`

Reply:
Doesn't need to, its only used when you want to have a default set. I would agree thought that across the mono-repos that this is inconsistently applied. I've tweaked it nonetheless :)